packages/adapter/addon/rest.js

```diff
@@ -1067,9 +1063,28 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
   },
 
   _ajax(options) {
-    if (get(this, 'useFetch')) {
+    if (this.useFetch) {
       this._fetchRequest(options);
-    } else if (get(this, 'fastboot.isFastBoot')) {
+    } else if (DEPRECATE_NAJAX && get(this, 'fastboot.isFastBoot')) {
+      if (has('fetch')) {
+        deprecate(
+          'You have ember-fetch and jquery installed. To use ember-fetch, set `useFetch: true` in your adapter.  In 4.0, ember-data will fallback to ember-fetch instead of najax when both ember-fetch and jquery are installed in FastBoot.',
+          false,
+          {
+            id: 'ember-data:najax-fallback',
+            until: '4.0',
+          }
+        );
+      } else {
+        deprecate(
+          'In 4.0, ember-data will default to ember-fetch instead of najax in FastBoot.  It is recommended that you install ember-fetch or similar as a fetch polyfill in FastBoot.',
+          false,
+          {
+            id: 'ember-data:najax-fallback',
+            until: '4.0',
+          }
+        );
```

Comment:
Hmmm, maybe we should move this deprecation to happen in the `useFetch` computed and funnel everything through that still? The thing I'm concerned with is a massive flood of deprecations to the console that are all exactly the same. AFAICT, we are going to trigger this deprecation **every** AJAX request we need to make...